### PR TITLE
ci: basic validation

### PR DESCRIPTION
**Pull request by firelizzard18**
Add basic validation to CI via CLI

### CODE REVIEW

**Comment by firelizzard18** [comment]
@bunfield, would you look over the CLI change? @ilzheev, would you look over the `validate` job, in case there's anything else you think I should add?

File: cmd/cli/cmd/util.go [hunk 1]
```diff
@@ -246,21 +246,22 @@ type ActionResponse struct {
...
+		if ok {
 			a.Code = "ok"
 		}
 		dump, err := json.Marshal(a)
 		if err != nil {
 			log.Fatal(err)
 		}
 		log.SetFlags(log.Flags() &^ (log.Ldate | log.Ltime))
-		log.Fatal(string(dump))
+		log.Print(string(dump))
```
Comment (bunfield) [inline cmd/cli/cmd/util.go:259]: This is OBE by AC-611 but I'll approve and update AC-611 PR.

File: cmd/cli/cmd/util.go [hunk 2]
```diff
@@ -274,7 +275,11 @@ func (a *ActionResponse) Print() {
 		if a.Codespace != "" {
 			out += fmt.Sprintf("\tCodespace\t\t:\t%s\n", a.Codespace)
 		}
-		log.Fatal(out)
+		log.Print(out)
+	}
+
+	if !ok {
+		os.Exit(1)
```
Comment (bunfield) [inline cmd/cli/cmd/util.go:282]: same comment as above.

**Review by bunfield — APPROVED** [review]

File: cmd/cli/cmd/util.go (hunk 1 above)
Comment (firelizzard18) [reply cmd/cli/cmd/util.go:259]: :+1:

**Pull request merged**